src/Services/JiraSyncWorkflow.php

```
@@ -0,0 +1,275 @@
+            ),
+        ];
+    }
+
+    /**
+     * @param array<string, mixed> $integration
+     * @return array{ok: true, message: string}
+     */
+    public function bulkPullStatus(array $integration): array
+    {
+        $integrationId = (int) $integration['id'];
+        $mappings = SyncMapping::findByIntegration($this->db, $integrationId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\SyncMapping' in method 'bulkPullStatus'. (undefined)

(StaticAccess)